test/python/test_quantumprogram.py

```diff
@@ -1132,16 +1132,16 @@ def test_simulator_online_size(self, qe_token, qe_url):
         """
```

Comment:
I'm not sure about this, according to the test and parameters, the test wants to check if there is no response for huge circuits using the standard timeout. This patch changes the purpose of the test: now it is testing that it raises a `JobTimeoutError` for unmeeting waitings.